examples/all-clusters-app/esp32/main/Kconfig.projbuild

```diff
@@ -103,10 +103,11 @@ menu "Demo"
 endmenu
 
 menu "PW RPC Debug channel"
+depends on ENABLE_PW_RPC
     config EXAMPLE_UART_PORT_NUM
         int "UART port number"
         range 0 2 if IDF_TARGET_ESP32
-        default 0 if IDF_TARGET_ESP32
+        default 0
         help
             UART communication port number for the example.
             See UART documentation for available port numbers.
```

Comment:
You will also have the define the range of UART Tx and Rx pins for ESP32-C3 and update `EXAMPLE_UART_RXD` and `EXAMPLE_UART_TXD` for all these examples accordingly

Reply:
Done